src/main/java/com/onevizion/scmdb/facade/DbScriptFacade.java

```diff
@@ -159,6 +158,17 @@ public void deleteAll(Collection<SqlScript> scripts) {
                                            .collect(Collectors.toList()));
     }
 
+    public List <SqlScript> getDbScriptsWithText(List<SqlScript> scripts) {
+        Set<Long> ids = new HashSet<>();
```

Comment:
method's code can be simplified
<img width="980" alt="image" src="https://user-images.githubusercontent.com/3282060/210518594-888ec21f-4720-441f-ad51-a8aa14ac56f3.png">

Reply:
this method is removed